docs/multidatacenter.md

```diff
@@ -14,9 +14,13 @@ rack awareness.
 
 ## Kubernetes cluster prerequisites
 
-At this time, KUDO Cassandra needs a single Kubernetes cluster spanning all the
-datacenters. A Cassandra cluster running on two or more Kubernetes clusters is
-not supported at the moment
+### Naming
+
+Cassandra datacenters can either run in a single Kubernetes cluster that is
```

Comment:
```suggestion
Cassandra clusters can either run in a single Kubernetes cluster that is
```
?

Reply:
Mutiple Datacenters form a single Cassandra cluster. At least that's what I'm trying to say here. I'll add another sentence to distinguish between Cassandra cluster and datacenter.